mappings/net/minecraft/entity/data/TrackedDataHandlerRegistry.mapping

```diff
@@ -35,3 +37,4 @@ CLASS net/minecraft/unmapped/C_fegtripb net/minecraft/entity/data/TrackedDataHan
 		ARG 0 handler
 	METHOD m_goylrcty get (I)Lnet/minecraft/unmapped/C_xrzqxzbg;
 		ARG 0 id
+	CLASS C_aaxqothd
```

Comment:
Empty mapping

Reply:
lost the contents dropping invalid mappings, I'll re-map the class tonight

Reply:
turns out the class just fully disappeared